drivers/hmis/spec/requests/hmis/update_project_spec.rb

```diff
@@ -24,8 +24,8 @@
   let(:test_input) do
     {
       project_name: 'Project 1',
-      operating_start_date: Date.today.strftime('%Y-%m-%d'),
-      operating_end_date: (Date.today - 1.day).strftime('%Y-%m-%d'),
+      operating_start_date: (Date.today - 1.day).strftime('%Y-%m-%d'),
+      operating_end_date: Date.today.strftime('%Y-%m-%d'),
```

Comment:
This is probably fine, but you may want to span more than one day, sometimes the timezone gets in the way when tests are run right around the turn over.  Also, `Date.current` takes into account the timezone which is usually what you want.
https://stackoverflow.com/questions/6635363/what-is-the-difference-between-date-current-and-date-today
```suggestion
      operating_start_date: (Date.yesterday).strftime('%Y-%m-%d'),
      operating_end_date: Date.current.strftime('%Y-%m-%d'),
```

Looks like Date.yesterday correctly calls `Date.current.yesterday`  to include the timezone